test/jdk/java/lang/reflect/AccessFlag/ClassAccessFlagTest.java

```diff
@@ -25,12 +25,18 @@
  * @test
  * @bug 8266670 8291734 8296743
  * @summary Test expected AccessFlag's on classes.
+ * @modules java.base/jdk.internal.misc
+ * @enablePreview
```

Comment:
Should drop `@enablePreview`.  Same as `test/jdk/java/lang/reflect/AccessFlag/BasicAccessFlagTest.java`